rs/ic_os/fstrim_tool/src/lib.rs

```diff
@@ -85,19 +85,34 @@ fn write_initialized_metrics_if_not_exist(metrics_filename: &str) -> Result<()>
     write_metrics_using_tmp_file(&metrics, metrics_filename)
 }
 
+fn is_node_assigned() -> bool {
+    Path::new("/var/lib/ic/data/cups/cup.types.v1.CatchUpPackage.pb").exists()
+}
```

Comment:
This is a pretty critical check, but it looks OK to me as long as some other team can ensure we are OK to clear the rest of data as long as this file does not exist.

Reply:
Indeed, the file is created when a node is assigned to a subnet, and removed once the node is unassigned. This is checked as part of the `node_reassignment_test` in the assertion linked above, which would fail if we ever changed the file path.

Reply:
Thanks!

Reply:
Sorry for being late to the party, I only saw this PR now!

Would it make sense to extract the path & filename of the CUP to a `const` that would be used both here, and in the consensus tests referenced above? Then at least there would be a lower probability of having one changed without the other.

As its implemented now, IIUC, if the location of the CUP changes, `fstrim_tool` would always interpret the node as being unassigned, and run `fstrim` also on the data directory once per week (as long as the data directory is non-empty). That shouldn't be a problem, but I guess the reason for having the check here in the first place is to reduce the risk.